CMakeLists.txt

```diff
@@ -715,9 +718,6 @@ if (NOT (MINGW OR APPLE OR FREEBSD OR OPENBSD OR DRAGONFLY))
 add_compile_options("${WARNINGS_AS_ERRORS_FLAG}") # applies only to targets that follow
```

Comment:
That's the issue.
With this change, -Werror will again fail to be passed for src/ and contrib/. And, -Werror will be passed to tests, which still have warnings (perhaps no longer true?) which will fail the build.
So, the fix is to move -Werror setting into src/CMakeLists.txt and contrib/CMakeLists.txt.

Reply:
I did not realize this. I also don't recall warnings in the tests, but then I use GCC, which is typically less noisy than CLANG.